packages/dapp/pages/chain/[chainId]/[address]/index.tsx

```diff
@@ -712,80 +608,6 @@ const QuestChainPage: React.FC<Props> = ({ questChain: inputQuestChain }) => {
           )}
         </VStack>
       </SimpleGrid>
-
-      <Modal isOpen={!!quest && isOpen} onClose={onModalClose} size="xl">
```

Comment:
I'm concerned about moving the Modal into the component which is inside a loop. That means one modal is rendered for each quest, when we could just have one component and replace the text for each.
For now though I'm merging this and can revisit if we find any performance issues.